src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -105,22 +105,22 @@ default int checkMatches(int expectedMatches, double waitFor, double timeTook) {
      * @param timeTook      - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: the css value of the attribute passed in. null will be returned if the element's css isn't accessible
      */
-    default String checkCssValue(String attribute, String expectedValue, double waitFor, double timeTook) {
+    String checkCssValue(String attribute, String expectedValue, double waitFor, double timeTook) {
         // get the actual css element value
-        String elementCssValue = getElement().get().css(attribute);
+        String elementCssValue = this.element.get().css(attribute);
         // record the result
         if (elementCssValue == null) {
-            getReporter().fail(getElement().prettyOutput() + " having a css attribute of <i>" + attribute +
-                    "</i> with a value of <b>" + expectedValue + "</b>", waitFor, "Unable to assess the css of " + getElement().prettyOutputEnd().trim(), timeTook);
+            this.reporter.fail(this.element.prettyOutput() + " having a css attribute of <i>" + attribute +
+                    "</i> with a value of <b>" + expectedValue + "</b>", waitFor, "Unable to assess the css of " + this.element.prettyOutputEnd().trim(), timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</i> with a value of <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)